src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -6998,6 +6998,61 @@ class BodyVisitor : public CommonVisitor<BodyVisitor> {
                 tmp = ASR::make_LogicalConstant_t(al, loc, is_lower,
                         ASRUtils::TYPE(ASR::make_Logical_t(al, loc, 4)));
                 return;
+            } else if(attr_name == "isalpha") {
+                /*
+                * Specification:
+                * Return True if all characters in the string are alphabetic (letters), and False otherwise.
+                */
+               bool is_alpha = true;
+                for (auto &i : s_var) {
+                    int ch_ord = static_cast<int>(i);
+                    if ((ch_ord >= 65 && ch_ord <= 90) || (ch_ord >= 97 && ch_ord <= 122)) {
+                        continue;
+                    }
+                    is_alpha = false;
```

Comment:
Can we use `std::isalpha()` in this `if` condition? Also, I would inverse the `if` condition and add a `break` statement inside it. For example something like:

```cpp
if (!std::isalpha(i)) {
    is_alpha = false;
    break;
}
```